packages/svelte-ux/src/lib/components/LanguageSelect.svelte

```diff
@@ -0,0 +1,49 @@
+<script lang="ts">
+  import Button from './Button.svelte';
+  import Menu from './Menu.svelte';
+  import MenuItem from './MenuItem.svelte';
+  import { cls } from '../utils/styles';
+  import { getSettings } from './settings';
+  const { locale } = getSettings();
+
+  let open = false;
+
+  type Language = {
+    name: string;
+    code: string;
+    flag: string;
+  };
+
+  export let languagesDemo: Language[] = [
+    { name: 'English', code: 'en', flag: '🇺🇸' },
+    { name: 'Français', code: 'fr', flag: '🇫🇷' },
+    // add more for the demo
+  ];
+  $: languageSelected = languagesDemo.find((c) => c.code === $locale)!;
+</script>
+
+<Button on:click={() => (open = !open)}>
```

Comment:
[MenuButton](https://format-last-qol-for-next.svelte-ux.pages.dev/docs/components/MenuButton) could simplify this a good bit, although we'd probably need to access a slot for the note about affecting dates and numbers like the `actions` slot of `SelectField`).  Probably not worth changing now, just mentioning it if you hadn't seen it.  The `stepper` functionality` is rather nice (and I've added `stepper` to `SelectField` in my current theme generator PR)

Reply:
Thx for the info. 
I copy / pasted the `<ThemeSelect />` as a starting point, that's why ^^